fpdf/fpdf.py

```diff
@@ -2235,16 +2236,6 @@ def add_font(
                 " this feature is deprecated since v2.5.1 and has been removed in v2.5.3."
             )
 
-        if uni != "DEPRECATED":
-            warnings.warn(
-                (
-                    '"uni" parameter is deprecated since v2.5.1, '
```

Comment:
By introducing `deprecated_parameter()`, we lose the information of the deprecation version.

Could we maybe preserve it by passing it to `deprecated_parameter()`?